# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mac-fdisk/larger-bootstrap.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mac-fdisk-0.1.orig/pdisk.c.vanilla	2022-05-25 10:38:37.322914578 +0200
+++ mac-fdisk-0.1.orig/pdisk.c	2022-05-25 10:40:26.562920214 +0200
@@ -354,7 +354,7 @@
 	    printf("  P    (print ordered by base address)\n");
 	    printf("  i    initialize partition map\n");
 	    printf("  s    change size of partition map\n");
-	    printf("  b    create new 800K bootstrap partition\n");
+	    printf("  b    create new bootstrap partition\n");
 	    printf("  c    create new Linux partition\n");
 	    printf("  C    (create with type also specified)\n");
 	    printf("  d    delete a partition\n");
@@ -498,8 +498,8 @@
 	return;
     }
 
-    // create 800K type Apple_Bootstrap partition named `bootstrap'
-    add_partition_to_map(kBootstrapName, kBootstrapType, base, 1600, map);
+    // create 2048K type Apple_Bootstrap partition named `bootstrap'
+    add_partition_to_map(kBootstrapName, kBootstrapType, base, 4096, map);
 }
 
 
--- mac-fdisk-0.1.orig/mac-fdisk.8.in.vanilla	2022-05-25 12:20:21.701229498 +0200
+++ mac-fdisk-0.1.orig/mac-fdisk.8.in	2022-05-25 12:20:41.643230527 +0200
@@ -83,7 +83,7 @@
 P    print ordered by base address
 i    initialize the partition map
 s    change size of partition map
-b    create new 800K Apple_Bootstrap partition (used by yaboot)
+b    create new Apple_Bootstrap partition (for grub or yaboot)
 c    create new standard Linux type partition
 C    create new partition, specifying the partition type
 d    delete a partition
@@ -147,7 +147,7 @@
 The
 .B b
 command is a shortcut to create the bootstrap partition used on PowerPC
-NewWorld machines. It is equivalent to creating an 800k
+NewWorld machines. It is equivalent to creating an 2MB
 .B Apple_Bootstrap
 type partition using the 
 .B C

